integration-libs/opf/base/root/config/opf-config.ts

```diff
@@ -15,6 +15,26 @@ export abstract class OpfConfig {
   opf?: {
     opfBaseUrl?: string;
     commerceCloudPublicKey?: string;
+    paymentOption: {
+      /**
+       * The map of info message translation keys for specific payment configuration IDs
+       * @example
+       * ```ts
+       * const opfConfig = {
+       *   opf: {
+       *     paymentInfoMessagesMap: {
+       *       213: 'opfCheckout.payPalPaymentInfoMessage' // Message key for payment method ID 213
+       *     }
+       *   }
+       * };
+       * ```
+       */
+      paymentInfoMessagesMap?: Record<number, string>;
+      /**
+       * Enables the payment info message section within the payment options
+       */
+      enableInfoMessage?: boolean;
```

Comment:
this flag value is not read anywhere, shouldn't be it use in template or component on condition to display the InfoMessage?

Reply:
changed